cmd/release/number/internal/number_pr.go

```diff
@@ -0,0 +1,48 @@
+package internal
+
+import (
+	. "../../internal"
```

Comment:
Can we make this absolute paths instead of relative paths like here?

Reply:
It really didn't like it when I tried to do this. I'm planning to switch to go modules at some point. Maybe this would help?